examples/cryptocurrency/examples/demo.rs

```diff
@@ -64,10 +64,14 @@ fn node_config() -> NodeConfig {
 fn main() {
     exonum::helpers::init_logger().unwrap();
 
+    let external_runtimes: Vec<(u32, Box<dyn exonum::runtime::Runtime>)> = vec![];
```

Comment:
I think we should refine `Node` and `Blockchain` creation algorithm and enforce builders usage.

So we will have a much more cleaner code and encapsulate the `Node` internal structure:
```rust
let node = NodeBuilder::new()
    .with_db(TemporaryDB::new()
    .with_service(CryptocurrencyService)
    .finalize(node_config());
```

Reply:
Good idea. 👍🏻